code/modules/mob/delays.dm

```diff
@@ -5,7 +5,7 @@
 // Reduces duplicated code by quite a bit.
 /datum/delay_controller
 	// Delay clamps (for adminbus, effects)
-	var/min_delay = 1
+	var/min_delay = 0.001
```

Comment:
Pretty sure this can't be lower than tick_lag, otherwise mobs will start teleporting when they move. 

Remember that period of time really early when I was playing with tick lag on the server and people would say they literally could not move less than 2 tiles at once no matter how careful they were? It was due to the fact that if the delayer is less than tick lag you will start teleporting more than one tile and it's fucking ANNOYING.